docs/spec/manifest-v2-2.md

```diff
@@ -0,0 +1,109 @@
+# Distribution Content Manifests
+
+A Content Manifest is a simple JSON file which contains general fields that
+describe an object and its dependencies. The goal is for these manifests to
+describe a reference to any application data, optional metadata or labels, and
+to reference any dependent objects in a content-addressable and verifiable way.
+These manifests can also be stored in a repository and referenced by digest.
+
+## Field Descriptions
+
+- **`schemaVersion`** *int*
+
+    Specifies the schema version of the manifest as an integer. This document 
+    describes version `2` only.
+
+- **`target`** *object*
+
+    The target field may describe *any* object stored in a repository, allowing
```

Comment:
@jlhawn @stevvooe in a previous revision of this proposal, I believe the target could point at external objects as well as collocated objects. If I'm reading this current version correctly, references to external objects are no longer allowed?